tests/e2e/ctl_v3_kv_test.go

```diff
@@ -22,28 +22,14 @@ import (
 	"go.etcd.io/etcd/tests/v3/framework/e2e"
 )
 
-func TestCtlV3PutNoTLS(t *testing.T)     { testCtl(t, putTest, withCfg(*e2e.NewConfigNoTLS())) }
```

Comment:
@serathius I am curious why are we removing bunch of test functions? Thanks!

Reply:
I believe the tests are moved into [tests/common/kv_test.go](https://github.com/etcd-io/etcd/pull/13740/files#diff-984b74501969e855745d40f470d8848b804927baabc7cb7a3dd4d9e43c7b93a9).

Reply:
Yes, this are exact tests that are migrated.

Reply:
Ok, cool. Thanks both!!